package.json

```diff
@@ -42,7 +41,8 @@
   },
   "peerDependencies": {
     "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react-dom": "^0.14.0",
+    "react-tap-event-plugin": "^0.2.0"
```

Comment:
Thanks for the response, @oliviertassinari !
Sorry, my fault, it's not a dependency of event-plugin.
I mean, previously,  after `npm install`,  the directory structure  is
```bash
➜  webpack-example git:(master) ✗ ls node_modules/material-ui/node_modules
history               lodash.debounce       warning
inline-style-prefixer lodash.throttle
```
Normally only packages in the `dependencies` can be installed here. Not sure why there is a `history` package installed here.